### ISSUE TITLE
Persistent clan invitations

### ISSUE THREAD

**Issue by cmyui** [osuAkatsuki/bancho.py#199]
As a clan owner, I'd like to be able to
- create persistent invitation links
- send these invitations to other users
- specify which clan roles are able to invite others (owner, officers, members)
- manage the existing set of pending invitations

**Comment by memsdm05** [comment]
new db table with autoincrement ids for each invitation. Works like discord’s server invitation (you can limit you can join and how long it lasts)

**Comment by cmyui** [comment]
> new db table with autoincrement ids for each invitation. Works like discord’s server invitation (you can limit you can join and how long it lasts)

yep, may want a hash for the actual invite code itself. might be doable in pure sql?

**Comment by memsdm05** [comment]
on second thought invites should be a random string id to prevent iterating over a bunch of ids

**Comment by cmyui** [comment]
yup, that's what i was getting at with the hash
